Modules/services/Controller/ServicesprojectsController.php

```diff
@@ -686,6 +706,33 @@ public function getTaskFileAction($id_space, $id_task) {
         $this->render(['data' => $file]);
     }
 
+    public function openFileAction($id_space, $id_task) {
+        $this->checkAuthorizationMenuSpace("services", $id_space, $_SESSION["id_user"]);
+        $taskModel = new SeTask();
+        $task = $taskModel->getById($id_space, $id_task);
+
+        // If private task, check if user is the owner of the task or if user is at least manager
+        if ($task['private'] == 1
+            && (!$this->role >= CoreSpace::$MANAGER || $task['id_owner'] != $_SESSION["id_user"])) {
```

Comment:
invalid test I think   if not manager or not owner , the "not owner" will match  if i am a manager but not the owner

if .... && ( $this->role < CoreSpace::$MANAGER && $task['id_owner'] != $_SESSION["id_user"] )

no?

Reply:
ah yeah, thx